lib/graphQl/index.js

```diff
@@ -13,10 +13,12 @@ const writeTypes = require('./writeTypes.js')
 jsonApiGraphQL.with = app => {
   const config = jsonApi._apiConfig
 
-  app.use(new RegExp(`${config.base}$`), graphqlHTTP({
-    schema: jsonApiGraphQL.generate(jsonApi._resources),
-    graphiql: !!config.graphiql
-  }))
+  if (config.graphiql !== false) {
```

Comment:
What we were suggesting was a *new* configuration property named `strict`, but we can get round to doing that in a later PR.